doc/catalog-format-spec.md

```diff
@@ -508,6 +508,63 @@ static, i.e. they cannot depend on the context.
    crate_2.var1 = "Debug"
    ```
 
+## Work-in-progress dependency overrides
+
+It is usual to develop several interdependent crates at the same time. In this scenario, it is often impractical to rely on indexed releases which are not intended to be modified. Instead, one would prefer to use a work-in-progress version of a crate to fulfill some dependency.
+
+Alire provides *pins* to support this use case. Pins are overrides to dependencies, are intended to be used locally, and to be fulfilled by proper dependencies once a crate is ready to be published. The use of pins is based on two ideas:
```

Comment:
This sentence reads strange to me. I would suggest:
`Pins override dependencies, they are intended to be used locally, and to be fulfilled by proper dependencies once a crate is ready to be published.`